src/ChainlinkVRFV2Direct.sol

```diff
@@ -155,6 +152,12 @@ contract ChainlinkVRFV2Direct is VRFV2WrapperConsumerBase, Ownable, RNGInterface
     return _requestConfirmations;
   }
 
+  /// @notice Returns the VRF V2 Wrapper contract that this contract uses.
+  /// @return The VRFV2Wrapper contract
+  function vrfV2Wrapper() external view returns (VRFV2Wrapper) {
+    return VRFV2Wrapper(address(VRF_V2_WRAPPER));
```

Comment:
`VRF_V2_WRAPPER` lives in `VRFV2WrapperConsumerBase` as an internal constant?

Reply:
Good point...any address constants we should be concerned about?  differ between mainnet + optimism etc?

Reply:
I don't know why it's an internal constant in the wrapper, but it is. What's the concern here?